cmd/users/main.go

```diff
@@ -99,7 +139,7 @@ func main() {
 	dbTracer, dbCloser := initJaeger("users_db", cfg.jaegerURL, logger)
 	defer dbCloser.Close()
 
-	svc := newService(db, dbTracer, cfg.secret, logger)
+	svc := newService(db, dbTracer, cfg.secret, cfg.resetURL, cfg, logger)
```

Comment:
If you are passing `cfg` as argument it's redundant to also pass  `cfg.secret, cfg.resetURL`